### PR TITLE
hu.kramo.Cartridges

### PR DESCRIPTION
**Pull request by kra-mo**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [ ] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*

The `org.freedesktop.Flatpak` talk is required for users to launch games on the host system.

- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: hu.kramo.Cartridges.json [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://files.pythonhosted.org/packages/36/2b/61d51a2c4f25ef062ae3f74576b01638bebad5e045f747ff12643df63844/PyYAML-6.0.tar.gz",
+                    "sha256": "68fb519c14306fec9720a2a5b45bc9f0c8d1b9c72adf45c37baedfcd949c35a2"
+                }
+            ]
+        },
+        {
+            "name" : "blueprint-compiler",
```
Comment (hfiguiere) [inline hu.kramo.Cartridges.json:49]: you need to clear this build-only dependency.

File: hu.kramo.Cartridges.json [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+                    "url" : "https://github.com/kra-mo/cartridges",
+                    "tag": "v1.0",
+                    "commit": "501747ee363f2daa9cb7a725ef1b46254ea62eba"
+                }
+            ]
+        }
+    ],
+    "build-options" : {
+        "env" : {        }
+    }
```
Comment (hfiguiere) [inline hu.kramo.Cartridges.json:75]:
```suggestion
    ]
```

File: hu.kramo.Cartridges.json [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        "--socket=wayland",
+        "--talk-name=org.freedesktop.Flatpak",
+        "--talk-name=org.gtk.vfs.*",
+        "--filesystem=xdg-run/gvfsd",
+        "--filesystem=~/.steam/steam/:ro",
+        "--filesystem=xdg-config/heroic/:ro",
+        "--filesystem=xdg-data/bottles/:ro",
+        "--filesystem=~/.var/app/com.valvesoftware.Steam/data/Steam/:ro",
+        "--filesystem=~/.var/app/com.heroicgameslauncher.hgl/config/heroic/:ro",
+        "--filesystem=~/.var/app/com.usebottles.bottles/data/bottles/:ro"
```
Comment (hfiguiere) [inline hu.kramo.Cartridges.json:20]: that's a lot of permissions.

**Comment by barthalion** [comment]
bot, build hu.kramo.Cartridges

**Comment by hfiguiere** [comment]
Build relies on https://github.com/flathub/flatpak-builder-lint/pull/128

**Comment by barthalion** [comment]
bot, build hu.kramo.Cartridges

**Pull request closed without merging**